src/org/jitsi/meet/test/MaxUsersTests.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright @ 2015 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.meet.test;
+
+
+import org.jitsi.meet.test.util.*;
+import org.openqa.selenium.*;
+
+import junit.framework.*;
+
+/**
+ * Test for max users prosody module.
+ * 
+ * NOTE: The tests assumes that the module is deployed and configured.
+ *
+ * @author Hristo Terezov
+ */
+public class MaxUsersTests
+    extends TestCase
+{
+    /**
+     * Number of participants in the call.
+     */
+    public static int MAX_USERS = 3;
```

Comment:
It turns out that the plugin also counts jicofo as an user(which is fine). So when we use 3 it turns out that the max users limit is actually 2 and when you start the third one it will fail. I did not get that when reading the code. It would be more readable if some calculation would include 1 for jicofo and 1 for the user that will exceed the limit when constructing the length of "participants" array in the code below.